pkg/network/render.go

```diff
@@ -185,9 +213,9 @@ func RenderAdditionalNetworks(conf *netv1.NetworkConfigSpec, manifestDir string)
 		return nil, errors.Errorf("invalid Additional Network Configuration: %v", errs)
 	}
 
-	// render Multus when additional networks is provided
+	// render the CRD when additional networks are provided
```

Comment:
Shall we also capture config error that when additionalNetworks are provided but DeployMultus is set to false?

Reply:
I was actually intending that to be supported, in case some 3rd party plugin also implements the CRD spec but does not use Multus. Again, I think AdditionalNetworks (eg the NPWG spec CRs) and Multus (an implementation of the NPWG spec) should be separate things.